assets/js/modules/reader-revenue-manager/components/setup/SetupMain.stories.js

```diff
@@ -27,8 +32,27 @@ function Template() {
 
 export const Default = Template.bind( {} );
 Default.storyName = 'Default';
+Default.scenario = {
+	label: 'Modules/ReaderRevenueManager/Setup/SetupMain/Default',
+	delay: 250,
```

Comment:
We already have a base `delay` in our backstop config which is much more – does this actually help or is it just copied?

Reply:
Hmm - fair question. The story does have a brief loading state so I thought we might need the delay. But, maybe the default will suffice. I've removed the delay, let's see how it fares.

Reply:
To be honest, I had forgotten the default is 1 second which as you say is much more and will probably be enough.